lib/cli/interactive-setup/console-login.js

```diff
@@ -6,13 +6,20 @@ const promptWithHistory = require('@serverless/utils/inquirer/prompt-with-histor
 const login = require('../../commands/login/console');
 const { showOnboardingWelcome } = require('./utils');
 
-const loginOrRegisterQuestion = async ({ stepHistory }) =>
-  promptWithHistory({
-    message: 'Do you want to login/register to Serverless Console?',
+const loginOrRegisterQuestion = async (context) => {
+  let message;
+  if (context.initial.isInServiceContext) {
+    message = 'Press [Enter] to create a free Serverless Console account';
+  } else {
+    message = 'Do you want to login/register to Serverless Console?';
+  }
```

Comment:
TL/DR to play it safe.

In more details:

- when creating a new project we don't display the "welcome message", so there's no introduction to what Console is, it feels much more forced
- when running `sls --console` in an existing project, the only possible action is onboarding into Console, so it's logical if we are a bit more direct (we ask for a confirmation, instead of a choice "do you want")
- when running `sls --console` in an empty directory, the main action is to create a new project, so it's more of a choice here ("do you want" makes more sense)

Here is a preview:

<img width="857" alt="Screen-000483" src="https://user-images.githubusercontent.com/720328/171438015-1740fe8c-7ad4-4ac8-9b7d-5edda314f4ed.png">
<img width="857" alt="Screen-000484" src="https://user-images.githubusercontent.com/720328/171438007-17f71541-5404-4f3f-adf2-aee295965936.png">

The second one technically leaves the choice to the user, but it's much more implicit. It could get (legitimate) backslash from the community.

Reply:
Great thanks for explanations, definitely makes sense 👍